pkgs/applications/science/biology/macse/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, jre, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "macse";
+  version = "2.03";
+
+  src = fetchurl {
+    url = "https://bioweb.supagro.inra.fr/${pname}/releases/${pname}_v${version}.jar";
+    sha256 = "0jnjyz4f255glg37rawzdv4m6nfs7wfwc5dny7afvx4dz2sv4ssh";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  phases = "installPhase";
+
+  installPhase = ''
```

Comment:
Please add `runHook preInstall` and `runHook postInstall` to the `installPhase` when overriding it.